content/contributor/content/practice/cli.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Content API from the CLI"
+bookhidden: true
+url: "/contributor/content/practice/cli"
+---
+
+This practice demonstrates how to play with the content API in a terminal. We'll be using the Decentraland Foundation's instance at `peer.decentraland.org`, and all you need is the `curl` command-line tool.
+
+{{< info >}}
+You can use [`httpie`](https://github.com/httpie/httpie) or format responses with [`jq`](https://github.com/stedolan/jq) to get more readable JSONs with each request. In this guide, we'll show formatted documents for clarity.
+{{< /info >}}
+
+This is what we'll do:
+
+1. Query the status of the content server.
+2. Locate and download the [snapshot]({{< relref "snapshots" >}}) for a list of entities.
+3. Obtain the manifest for an entity.
+4. Download some of the entitiy's files.
+
+Let's begin by querying the server status using `/about`:
```

Comment:
```suggestion
Let's begin by querying the server status using `/about`:
```

"Every Decentraland-capable server is identified by a realm, realms implement the /about endpoint and its response enables the discovery of all required servers and configurations for that 'Realm'"

Reply:
I don't think the practice guide is the place to add this information, though I know it's missing from where it should be (probably the architecture overview with mentions in other sections).

Can we leave this as future work, or do you think it's vital?